x-pack/legacy/plugins/reporting/export_types/csv/server/execute_job.ts

```diff
@@ -5,25 +5,26 @@
  */
 
 import { i18n } from '@kbn/i18n';
+import { CSV_JOB_TYPE } from '../../../common/constants';
+import { cryptoFactory } from '../../../server/lib';
 import {
-  ExecuteJobFactory,
   ESQueueWorkerExecuteFn,
+  ExecuteJobFactory,
   FieldFormats,
+  Logger,
   ServerFacade,
 } from '../../../types';
-import { CSV_JOB_TYPE, PLUGIN_ID } from '../../../common/constants';
-import { cryptoFactory, LevelLogger } from '../../../server/lib';
 import { JobDocPayloadDiscoverCsv } from '../types';
-import { createGenerateCsv } from './lib/generate_csv';
 import { fieldFormatMapFactory } from './lib/field_format_map';
+import { createGenerateCsv } from './lib/generate_csv';
```

Comment:
Apologies, but this PR contains some automated changes of an IDE tool that organize the imports